codebase/src/cpp/syscommon/include/syscommon/concurrent/Semaphore.h

```diff
@@ -97,7 +97,7 @@ namespace syscommon
 			 * @throws InterruptedException if the thread was interrupted before the permit could 
 			 * be acquired.
 			 */
-			bool tryAcquire( unsigned long timeoutMillis ) throw ( InterruptedException );
+			bool tryAcquire( unsigned long timeoutMillis ) noexcept(false);
```

Comment:
Formatting of the `noexcept` clause should follow our usual convention e.g. `noexcept( false )`